content/en/logs/guide/azure-logging-guide.md

```diff
@@ -5,6 +5,8 @@ further_reading:
 - link: "/logs/explorer/"
   tag: "Documentation"
   text: "Learn how to explore your logs"
+aliases:
+- /integrations/azure/#log-collection
```

Comment:
I'm not sure redirecting from an anchor is going to work - testing this, it still goes to the original page: https://docs-staging.datadoghq.com/bryce/update-azure-links/integrations/azure/#log-collection

Reply:
Thanks for catching that, forgot to remove it from testing. Updated in [#3fa0049](https://github.com/DataDog/documentation/pull/18765/commits/3fa00495374adcff3f26e831d395d3051fe387df)